db/migrate/20220126062422_create_articles.rb

```diff
@@ -3,6 +3,7 @@ def change
     create_table :articles do |t|
       t.string :title
       t.text :body
+      t.references :user, null: false, foreign_key: true
```

Comment:
Do you think `null: false` is required?
I guess adding reference with foreign by defaults adds this :)
Just have a look and confirm

Reply:
yes I was getting some issues while migration when I added this references column. So I regenerated the model using generate command, that is why these properties are auto-filled.